src/python/pants/bin/pants_runner.py

```diff
@@ -35,7 +35,7 @@ class PantsRunner:
     args: List[str]
     env: Mapping[str, str]
 
-    # This could be a bootstrap option, but it's preferable to keep these very limited to make it
```

Comment:
I'm fine with the change, but this is not an error. "It's" is a contraction of "it is", and so leaving it would have been fine too.